helm/values.yaml

```diff
@@ -209,7 +209,7 @@ argo-workflows:
       bucket: argo-workflows
 
       ## -- S3 endpoint
-      endpoint: "${RELEASE_NAME}-minio:9000"
+      endpoint: "{{ .Release.Name }}-minio:9000"
```

Comment:
I started gettign errors about not being able to access `http://${RELEASE_NAME}-minio:9000` - clearly `RELEASE_NAME` was not getting interpolated with an envvar. 

This is because the file is not preprocessed with `envsubst` like the other values files in `helm/examples/`. 

I didn't think this was doing anything either, because this is a `values.yaml` file - not a template. Templating shouldn't work here, but it does because I think the `argo-workflows` chart's templates use this endpoint value - and hence templating takes place. This is super confusing, because not every value you set supports templating - only those that get passed to a template.